Sources/Public/CalendarItemModel.swift

```diff
@@ -109,3 +110,16 @@ extension CalendarItemViewRepresentable {
   }
 
 }
+
+// MARK: SwiftUI Convenience Factory
+
+@available(iOS 13.0, *)
+extension View {
+
+  public var calendarItemModel: CalendarItemModel<SwiftUIWrapperView<Self>> {
+    CalendarItemModel<SwiftUIWrapperView<Self>>(
+      invariantViewProperties: .init(initialContent: self),
+      viewModel: .init(content: self))
+  }
```

Comment:
Might be useful to add some documentation comments, and detail the fact that this adds a UIHostingController as a child view controller of the closest VC so it doesn't require folks to look through the implementation to find out why the `UIViewController.children` changed

Reply:
Good call!